test/sanity/app-getproxyforurl/index.html

```diff
@@ -15,6 +15,14 @@
 	} else {
 		document.write('<h1 id="result">failure: expect "' + expect + '" but get ' + proxy +'</h1>');
 	}
+        nw.App.setProxyConfig('dummy:8080', 'http://foopy/proxy.pac');
+        var proxy = nw.App.getProxyForURL('http://www.example.com/');
+        var expect = 'DIRECT';
+        if (proxy === expect) {
+                document.write('<h1 id="result">success</h1>');
+        } else {
+                document.write('<h1 id="result">failure: expect "' + expect + '" but get ' + proxy +'</h1>');
+        }
 	</script>
```

Comment:
It can be passed when manually running the case, but failed with `python test/test.py -d ../../out/nw -t 60 sanity
`
> File "/home/fujunwei/largeharddisk/nwjs/src/content/nw/test/sanity/app-open-event/test.py", line 25, in <module> driver.quit()
> urllib2.HTTPError: HTTP Error 503: Service Unavailable